pkg/env/env_test.go

```diff
@@ -14,7 +14,7 @@ import (
 
 func TestInitConfig(t *testing.T) {
 	t.Parallel()
-	ConfigFile = "testdata/cpma-config.yml"
+	ConfigFile = "testdata/src/cpma-remote-config.yml"
```

Comment:
Why is `src` sub folder needed?

Reply:
Good note, it is not needed any more. I just had it, when was trying to generate a `config` file, and the `out` folder will be the location for the second one for comparison. Going to fix it now.